api/Comment.json

```diff
@@ -8,7 +8,7 @@
             "version_added": "1"
           },
           "chrome_android": {
-            "version_added": true
+            "version_added": "18"
```

Comment:
Can you test an instance for the basic support? This is the DOM node for a `<!-- comment -->` and I'm sure this was supported in Chrome 1.

Reply:
This line is for Chrome Android, actually!  Chrome Desktop is already set to 1, and the collector agrees.  :)

Reply:
Oh, I wasn't paying attention :) Thanks!